packages/server/tests/acceptance.spec.ts

```diff
@@ -256,157 +264,157 @@ describe('RPC Server Integration Tests', async function () {
+        const res = await utils.callSupportedRelayMethod('eth_submitWork', []);
         expect(res.data.result).to.be.equal(false);
     });
 
     it('should not support "eth_submitHashrate"', async function () {
-        await utils.callUnsupportedRelayMethod(this.relayClient, 'eth_submitHashrate', []);
-    });
-
-    it('should execute "eth_getBalance" for primary account', async function () {
-        const res = await utils.callSupportedRelayMethod(this.relayClient, 'eth_getBalance', [mirrorPrimaryAccount.evm_address, 'latest']);
-        expect(res.data.result).to.eq('0x1095793487d8e20c800');
+        await utils.callUnsupportedRelayMethod('eth_submitHashrate', []);
     });
 
-    it('should execute "eth_getBalance" for secondary account', async function () {
-        const res = await utils.callSupportedRelayMethod(this.relayClient, 'eth_getBalance', [mirrorSecondaryAccount.evm_address, 'latest']);
-        expect(res.data.result).to.eq('0x10f077b81e38c40a400');
+    it('should execute "eth_getBalance" for newly created account with 5000 HBAR', async function () {
+        const { accountInfo } = await utils.createEthCompatibleAccount(client, null, 5000);
+        const mirrorResponse = await utils.callMirrorNode(`accounts/${accountInfo.accountId}`);
+        const mirrorAccount = mirrorResponse.data;
+        const res = await utils.callSupportedRelayMethod('eth_getBalance', [mirrorAccount.evm_address, 'latest']);
+        expect(res.data.result).to.eq('0x10f0777f464e77a2400');
     });
 
     it('should execute "eth_getBalance" for non-existing address', async function () {
-        const res = await utils.callSupportedRelayMethod(this.relayClient, 'eth_getBalance', [nonExistingAddress, 'latest']);
+        const res = await utils.callSupportedRelayMethod('eth_getBalance', [nonExistingAddress, 'latest']);
         expect(res.data.result).to.eq('0x0');
     });
 
     it('should execute "eth_getBalance" for contract', async function () {
-        const res = await utils.callSupportedRelayMethod(this.relayClient, 'eth_getBalance', [mirrorContract.evm_address, 'latest']);
+        const res = await utils.callSupportedRelayMethod('eth_getBalance', [mirrorContract.evm_address, 'latest']);
         expect(res.data.result).to.eq('0x56bc75e2d63100000');
     });
 
     it('should execute "eth_getBalance" for account with id converted to evm_address', async function () {
-        const res = await utils.callSupportedRelayMethod(this.relayClient, 'eth_getBalance', [utils.idToEvmAddress(mirrorPrimaryAccount.account), 'latest']);
-        expect(res.data.result).to.eq('0x1095793487d8e20c800');
+        const { accountInfo } = await utils.createEthCompatibleAccount(client, null, 5000);
+        const accountId = accountInfo.accountId.toString();
+        const evmAddress = utils.idToEvmAddress(accountId);
+
+        // wait for the account to be imported in the Mirror Node
+        await utils.callMirrorNode(`accounts/${accountId}`);
```

Comment:
I think that instead of ` await aliasCreationResponse.getReceipt(client);` being executed within `createEthCompatibleAccount` we can do and wait for `getRecord`. After that we will know that the account has been created.. no need to `callMirrorNode` or something like that.